components/FilterButtons.vue

```diff
@@ -1,17 +1,34 @@
 <template>
-    <div>
-        <button>{{ $t("All") }}</button>
-        <button>{{ $t("Incomplete") }}</button>
-        <button>{{ $t("Complete") }}</button>
-    </div>
+  <div>
+    <button @click="showAll" :class="{ 'is-active': currentFilter === 'All' }">{{ $t("All") }}</button>
+    <button @click="showIncomplete" :class="{ 'is-active': currentFilter === 'Incomplete' }">{{ $t("Incomplete") }}</button>
+    <button @click="showComplete" :class="{ 'is-active': currentFilter === 'Complete' }">{{ $t("Complete") }}</button>
```

Comment:
instead of comparing values like this, use constants.

Reply:
resolved in later pr